### PR TITLE
Adding music service

### PR DESCRIPTION
**Pull request by zhexuanc**
- Adding music service to current master

### CODE REVIEW

File: lucida/musicservice/MusicConfig.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Music API configuration
+from pygn import Pygn
+
+# Service port number
+PORT = 8089
+
+# Music API username and password
+clientID = '173703779-EA19716F8D2A73DF7ECAF522D5050BF3' # Enter your Client ID from developer.gracenote.com here
```
Comment (saijel) [inline lucida/musicservice/MusicConfig.py]: @zhexuanc, we will need to remove this api key before merging to master. Have your own branch that across all the MRs applies your keys. Keep rebasing and applying that branch to master.

File: tools/start_all_tmux.sh [hunk 2]
```diff
@@ -50,6 +50,7 @@ declare -a imageclassification=("IMC" "$(pwd)/../lucida/djinntonic/imc/")
 declare -a digitrecognition=("DIG" "$(pwd)/../lucida/djinntonic/dig/")
 declare -a facerecognition=("FACE" "$(pwd)/../lucida/djinntonic/face")
 declare -a weather=("WE" "$(pwd)/../lucida/weather")
+declare -a musicservice=("MS" "$(pwd)/../lucida/musicservice")
 
 declare -a services=(
     commandcenter
```
Comment (saijel) [inline tools/start_all_tmux.sh]: @zhexuanc, need to add music service here as well to able to create the window for the music service.

Comment (zhexuanc) [inline tools/start_all_tmux.sh]: @saijel , thank you for correcting this and I fix it now!

File: lucida/musicservice/MusicConfig.py (hunk 1 above)
Comment (zhexuanc) [inline lucida/musicservice/MusicConfig.py]: @saijel, okay I have updated the PR now and remove the api key.

**Pull request merged**